docs/idx.md

```diff
@@ -361,6 +362,10 @@ If a `stateTokenExternalId` is not passed and a flow is not currently in progres
 
 Returns true if there is a saved transaction.
 
+#### `idx.poll`
+
+Resumes saved transaction and fires poll request to Okta API. Performs single poll request by default. In case `{ refresh: millisecondsInt` } parameter is passed, continues to poll until polling is considered complete by server or session times out. `refresh` value is available through the `nextStep` property of in-progress transaction.
```

Comment:
maybe only suggest to use the `refresh` param for browser env only? I feel use it on server side may cause more issues than the convenience it provides.